library/src/main/java/com/datatorrent/lib/formatter/Formatter.java

```diff
@@ -48,6 +49,13 @@
 {
   protected transient Class<?> clazz;
 
+  @AutoMetric
+  protected long errorTupleCount;
```

Comment:
Those need to be private. Have made it private and added protected  methods to access the values.